tests/message/test_concents.py

```diff
@@ -1,21 +1,19 @@
 # pylint: disable=protected-access
-from datetime import datetime, timedelta
 import math
 import time
 import unittest
+from datetime import datetime, timedelta
```

Comment:
Now it's not in alphabetical order. I preferred previous ordering.

Reply:
made by pycharm import optimizator, reverting